README.md

```diff
@@ -1 +1,63 @@
-# angular-what-is-a-custom-directive-readme
\ No newline at end of file
+# What is a custom Directive?
+
+## Overview
+
+One of the massive powerful aspects of Angular is the ability to actually be able to create our own directives.
+
+## Objectives
+
+- Describe what components are
+- Describe custom Directives
+- Create a custom Directive
+
+## Components
+
+Nowadays, most frontend applications are made up of small, reusable components. If we imagine a car, it would be made up with components such as wheels, windows, engine, etc.
+
+We can create these components using directives. We've already used directives that Angular gives us, but we can also make our own.
+
+Directives can both be HTML attributes and HTML elements. Most of our directives will be HTML elements, as they will have their own template inside of themselves instead of modifying the behaviour of existing elements.
+
+If we imagine an `ng-repeat`, that modifies the behaviour of the element it is placed on by repeating it for every item in an array.
```

Comment:
I think you can remove this line - it's hard to see the connection between this and the dropdown menu example below. Instead, maybe just mention that built-in angular directives often come in the form of HTML attributes and that they just generate HTML for us.